<launch>
    <node pkg='robotic_sas_auv_ros' type='node_accumulator.py' name='node_guidance' output='screen'/>
    <node pkg='robotic_sas_auv_ros' type='subs_teensy' name='node_accumulator' output='screen'/>
</launch>